index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>product landing page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+<header id="header">
+        <div id="title">
+            <img id="header-img" alt="HD logo " src="logo.png">
+        </div>
+    <nav id="nav-bar">
+        <a class="nav-link" href="#home">Home</a>
+        <a class="nav-link" href="#video_HD">Video</a>
+        <a class="nav-link" href="#about_us">About-Us</a>
+    </nav>
+</header>
+<main>
+<section id="home" >
+    <h1 class="well"> Welcome to HD Graphics Design</h1>
+    <img class="home" src="s.png" alt="HD Graphics" >
```

Comment:
- [ ] I think it would be better to insert in a div or display option to align the image center.

Reply:
* Giving clear feedback helps with effective communication, instead of asking to make a change only also add "why" the changes are necessary. See this [article on How to give a code review ](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md)for more details